tests/test_e2e.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import subprocess
+from kubernetes import client, config, utils
+
+
+@pytest.mark.e2e
+def test_deny_all_traffic_to_an_application():
+    namespace = "01-deny-all"
+    config.load_kube_config()
+    k8s_client = client.ApiClient()
+    corev1 = client.CoreV1Api()
+
+    corev1.create_namespace(client.V1Namespace(metadata=client.V1ObjectMeta(name=namespace)))
+    utils.create_from_yaml(k8s_client,
+                           "e2e-manifests/01-deny-all-traffic-to-an-application.yml",
+                           namespace=namespace)
+
+    # ToDo add sleep or wait until all resources are up otherwise we have a race condition
+    # ToDo handle execptions
+    res = subprocess.run(["illuminatio", "run", "--runner-image=localhost:5000/illuminatio-runner:dev"],
+                         capture_output=True,
```

Comment:
`capture_output` only works since python 3.7 (see https://docs.python.org/3.6/library/subprocess.html#subprocess.run for missing arg). This therefore fails for me using the setup from https://github.com/inovex/illuminatio/blob/master/docs/developing.md

Reply:
I just saw, that the pipeline only uses 3.7. This is fine IMO but should be stated somewhere

Reply:
Good Catch !